src/ImageGallery.tsx

```diff
@@ -38,6 +38,7 @@ const ImageGallery = (props: IProps & typeof defaultProps) => {
     thumbResizeMode,
     thumbSize,
     disableSwipe,
+    style,
```

Comment:
Sorry for the long wait. Could you add it also to the docs, please?

Reply:
Sure. I’ll update asap.